README.md

```diff
@@ -37,4 +39,36 @@ Steps of converting:
    Flags for location, plans, OS images are optional. 
    If you omit them empty and script can't find imported entities, 
    then product allows client to select necessary options manually during checkout.
-   We recommend you to check location, plan and OS image of converted products in Hostbill.
\ No newline at end of file
+   We recommend you to check location, plan and OS image of converted products in Hostbill.
+
+Step-by-step converting:
+1. Run `php index.php prepare` to load necessary data from SolusVM 2.
+   This step can take for example more than 5 minutes for loading 10000 servers.
+   Available flags:
+   * `--cluster-import` - ID of cluster import in SolusVM 2.
+
+   If you have more than one cluster imports in SolusVM 2, you can use this argument to point converter to necessary cluster import.
+   If you don't pass this argument, command get cluster import ID automatically.
+
+2. Run `php index.php copy-order-pages` to duplicate SolusVM 1 order page with its products to SolusVM 2 order page with converted products. Available flags:
+   * `--limit-group` - ID of limit group (required). Can be found in SolusVM 2 admin panel > Access > Limit Groups
+   * `--kvm-plan` - ID of KVM plan, needed if we can't find plan by plan name which is used in SolusVM 1 product.
+   * `--vz-plan` - ID of VZ plan, needed if we can't find plan by plan name which is used in SolusVM 1 product.
+   * `--vz-os-image` - ID of VZ OS image, needed if we can't find OS image by template which is used in SolusVM 1 product.
+   * `--kvm-os-image` - ID of KVM OS image, needed for KVM products because we don't import KVM OS images from SolusVM 1.
+   * `--location` - ID of location, needed if we can't find location by Node Group name which is used in SolusVM 1 product.
+   * `--role` - ID of role (optional, default value is ID of role with name "CLIENT")
+
+   Flags for location, plans, OS images are optional.
+   If you omit them empty and script can't find imported entities,
+   then product allows client to select necessary options manually during checkout.
+   We recommend you to check location, plan and OS image of converted products in Hostbill.
+
+3. (Optional) Run `php index.php dry-run` to see list of accounts to convert and revert converting command
+
+4. Run `php index.php convert-accounts` to convert SolusVM 1 accounts to SolusVM 2 accounts if its servers were imported
+
+5. Run `php index.php revert-accounts` to revert converting of converted to SolusVM 2 accounts to SolusVM 1. Available flags:
+   * `--accounts` - list of SolusVM 2 account ID`s to convert them back to SolusVM 1 account
```

Comment:
Add example for this command, will it be "php index.php revert-accounts --accounts "1,2,3"?